recorder/src/main/cpp/controller.cc

```diff
@@ -434,6 +447,7 @@ void Controller::issueWork(const std::string& host, const std::uint32_t port, st
                         populate_recording_header(rh, w, controller_id, controller_version);
                         writer->write_header(rh);
                     }
+                    s_v_working.update(1);
```

Comment:
Actually, I think this has accidentally survived, will kill it.

The work-type specific value gives us the same thing when summed.